common-lib/src/main/scala/com/gu/mediaservice/lib/ImageIngestOperations.scala

```diff
@@ -38,6 +38,10 @@ class ImageIngestOperations(imageBucket: String, thumbnailBucket: String, config
   private def storeOptimisedImage(storableImage: StorableOptimisedImage)
                        (implicit logMarker: LogMarker): Future[S3Object] =
     storeImage(imageBucket, optimisedPngKeyFromId(storableImage.id), storableImage.file, Some(storableImage.mimeType))
+  
+  def sendToQuarantine(id: String, file: File, mimeType: Option[MimeType], meta: Map[String, String] = Map.empty)
```

Comment:
I wonder whether this would better live in a `ImageQuarantineOperations` class. By separating this we can separate responsibilities and no longer need to configure the bucket in the Thrall service.